Packages/com.unity.inputsystem/InputSystem/Controls/InputControl.cs

```diff
@@ -790,6 +790,60 @@ protected void RefreshConfigurationIfNeeded()
             }
         }
 
+        /// <summary>
+        /// Refresh the configuration of the control. This is used to update the control's state (e.g. Keyboard Layout or display Name of Keys).
+        /// </summary>
+        /// <remarks>
+        /// The system will call this method automatically whenever change is made to one of the control's configuration properties.
+        /// See <seealso cref="RefreshConfigurationIfNeeded"/>.
+        /// </remarks>
+        /// <example>
+        /// <code>
+        /// public class MyDevice : InputDevice
```

Comment:
This example doesn't compile. You're likely missing  calling `using UnityEngine.InputSystem.*;`

I suggest copying this code into a .cs script file and seeing if at least compiles (and ideally works).

Same for other examples

Reply:
fixed in [04a3e1b](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/04a3e1bc8cb1e149b20c42555fb9a374ab97e908)